src/Development/IDE/LSP/CodeAction.hs

```diff
@@ -364,10 +370,6 @@ extendToWholeLineIfPossible contents range@Range{..} =
         extend = newlineAfter && _character _start == 0 -- takes up an entire line, so remove the whole line
     in if extend then Range _start (Position (_line _end + 1) 0) else range
 
--- | All the GHC extensions
-ghcExtensions :: Set.HashSet T.Text
-ghcExtensions = Set.fromList $ map (T.pack . show) ghcEnumerateExtensions
```

Comment:
The ghcEnumerateExtensions function is super awful, and only used in this one place. Can this patch now delete it?